docs/docs/how-to/manage_profiles.md

```diff
@@ -17,32 +17,29 @@ version: v1
 type: profile
 name: acme-github-profile
 context:
-  organization: ACME
-  group: Root Group
+  provider: github
 repository:
-  - context: github
-    rules:
-      - type: secret_scanning
-        def:
-          enabled: true
-      - type: branch_protection
-        params:
-          branch: main
-        def:
-          required_pull_request_reviews:
-            dismiss_stale_reviews: true
-            require_code_owner_reviews: true
-            required_approving_review_count: 1
-          required_linear_history: true
-          allow_force_pushes: false
-          allow_deletions: false
-          allow_fork_syncing: true
+  - type: secret_scanning
+    def:
+      enabled: true
+  - type: branch_protection
+    params:
+      branch: main
+    def:
+      required_pull_request_reviews:
+        dismiss_stale_reviews: true
+        require_code_owner_reviews: true
+        required_approving_review_count: 1
+      required_linear_history: true
+      allow_force_pushes: false
+      allow_deletions: false
+      allow_fork_syncing: true
```

Comment:
Is this branch protection rule still applicable? Thought we switched to the individual ones

Reply:
Ah, you're right 👍

Reply:
Done 👍